roles/osg/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Create osg role
+  os_keystone_role:
```

Comment:
Please wrap this in a `kolla_toolbox` invocation, these keystone modules are really annoying to use reliably on the host system; executing w/in the kolla_toolbox is by far the most reliable method I've found.